.gitignore

```diff
@@ -44,6 +44,7 @@ captures/
 .idea/assetWizardSettings.xml
 .idea/dictionaries
 .idea/libraries
+.idea/caches
```

Comment:
Already in the gitignore (two lines below)

Reply:
Thanks, git kept doing odd stuff with those files so I thought it wasn't there.